src/com/intellij/idea/plugin/hybris/runConfigurations/HybrisJUnitExtension.kt

```diff
@@ -71,6 +70,10 @@ class HybrisJUnitExtension : RunConfigurationExtension() {
 
         val modules: Array<Module> = ModuleManager.getInstance(project).modules
         for (module in modules) {
+            if (module.name.startsWith("CCv2")) {
```

Comment:
it may not be fully sufficient, please try to use `YFacet` instead to validate module type.

p.s. it may happen that end-user changed module name via `hybris4intellij.properties`